satpy/etc/readers/agri_l1.yaml

```diff
@@ -0,0 +1,327 @@
+
+datasets:
+  C01:
+    name: C01
+    wavelength: [0.45, 0.47, 0.49]
+    resolution:
+        1000: {file_type: agri_l1_1000m}
+        2000: {file_type: agri_l1_2000m}
+        4000: {file_type: agri_l1_4000m}
+    calibration:
+      reflectance:
+        standard_name: toa_bidirectional_reflectance
+        units: "%"
+      counts:
+        standard_name: counts
+        units: "1"
+    file_key: NOMChannel01
+    lut_key: CALChannel01
+    fill_value: 65535
+
+  C02:
+    name: C02
+    wavelength: [0.55, 0.65, 0.75]
+    resolution:
+        500: {file_type: agri_l1_0500m}
+        1000: {file_type: agri_l1_1000m}
+        2000: {file_type: agri_l1_2000m}
+        4000: {file_type: agri_l1_4000m}
+    calibration:
+      reflectance:
+        standard_name: toa_bidirectional_reflectance
+        units: "%"
+      counts:
+        standard_name: counts
+        units: "1"
+    file_key: NOMChannel02
+    lut_key: CALChannel02
+    fill_value: 65535
+
+  C03:
+    name: C03
+    wavelength: [0.75, 0.83, 0.90]
+    resolution:
+        1000: {file_type: agri_l1_1000m}
+        2000: {file_type: agri_l1_2000m}
+        4000: {file_type: agri_l1_4000m}
+    calibration:
+      reflectance:
+        standard_name: toa_bidirectional_reflectance
+        units: "%"
+      counts:
+        standard_name: counts
+        units: "1"
+    file_key: NOMChannel03
+    lut_key: CALChannel03
+    fill_value: 65535
+
+  C04:
+    name: C04
+    wavelength: [1.36, 1.37, 1.39]
+    resolution:
+        2000: {file_type: agri_l1_2000m}
+        4000: {file_type: agri_l1_4000m}
+    calibration:
+      reflectance:
+        standard_name: toa_bidirectional_reflectance
+        units: "%"
+      counts:
+        standard_name: counts
+        units: "1"
+    file_key: NOMChannel04
+    lut_key: CALChannel04
+    fill_value: 65535
+
+  C05:
+    name: C05
+    wavelength: [1.58, 1.61, 1.64]
+    resolution:
+        2000: {file_type: agri_l1_2000m}
+        4000: {file_type: agri_l1_4000m}
+    calibration:
+      reflectance:
+        standard_name: toa_bidirectional_reflectance
+        units: "%"
+      counts:
+        standard_name: counts
+        units: "1"
+    file_key: NOMChannel05
+    lut_key: CALChannel05
+    fill_value: 65535
+
+  C06:
+    name: C06
+    wavelength: [2.10, 2.22, 2.35]
+    resolution:
+        2000: {file_type: agri_l1_2000m}
+        4000: {file_type: agri_l1_4000m}
+    calibration:
+      reflectance:
+        standard_name: toa_bidirectional_reflectance
+        units: "%"
+      counts:
+        standard_name: counts
+        units: "1"
+    file_key: NOMChannel06
+    lut_key: CALChannel06
+    fill_value: 65535
+
+  C07:
+    name: C07
+    wavelength: [3.5, 3.72, 4.0]
+    resolution:
+        2000: {file_type: agri_l1_2000m}
+        4000: {file_type: agri_l1_4000m}
+    calibration:
+      brightness_temperature:
+        standard_name: toa_brightness_temperature
+        units: "K"
+      reflectance:

```

Comment:
C07-C13 provide both reflectance _and_ brightness temperature. I guess that should be brightness temperature only, right?

Reply:
@sfinkens Aha, thank you! it's actually `radiance`. Sorry for the mistake :(